crates/graphql/src/schema/objects/marketplace.rs

```diff
@@ -38,7 +38,7 @@ impl<'a> From<models::StoreConfigJson<'a>> for Marketplace {
             logo_url: logo_url.into_owned(),
             banner_url: banner_url.into_owned(),
             owner_address: owner_address.into_owned(),
-            auction_house_address: auction_house_address.into_owned(),
+            auction_house_address: auction_house_address.map(Cow::into_owned),
```

Comment:
We should have auction_house as is on the field but set to option and marked as deprecated then another field called `auction_houses` that returns the list of auction_houses associated to the marketplace.